### ISSUE TITLE
Quests in notes: what to do?

### ISSUE THREAD

**Issue by Sorxus** [We-the-People-civ4col-mod/Mod#393]
Currently you get some quests and in totes you can see about those quests.

![Quests-what to do](https://user-images.githubusercontent.com/37362600/102057534-55725880-3dee-11eb-951a-886794936230.JPG)

But there isn't specific info **what to do**. 

My suggestion is that if you hoover over quest there should be written "build 4 harbours" or "have 5 units hunters" etc.

**Comment by ErikSandbraaten** [comment]
I agree that this would be rather useful. It should be a simple matter of parsing the quest requirements (from their respective properties) and then displaying it somewhere.

**Comment by MrZorG33** [comment]
And one more thing: it would be nice to show how far you have already progressed in completing the quest.

**Comment by pedro-walter** [comment]
Since this issue was started more than two years ago, not sure if it's still relevant. Came here looking to suggest that we get quest progression, like the event that tells you that you have to trade X amount of goods with Europe, the current progress could also be displayed there. 

Should it be an issue on it's own or is it fine to be kept here? I could contribute to this with some pointers (no C pun intended).

**Comment by raystuttgart** [comment]
@raubwuerger Maybe this can be combined with what you want to do for "Trade Quests Screen".?
